doc/nrf/releases_and_maturity/releases/release-notes-2.7.99-cs1.rst

```diff
@@ -0,0 +1,110 @@
+
+For information on the included repositories and revisions, see `Repositories and revisions for v2.7.99-cs1`_.
+
+IDE and tool support
+********************
+
+`nRF Connect extension for Visual Studio Code <nRF Connect for Visual Studio Code_>`_ is the recommended IDE for |NCS| v2.7.99-cs1.
+See the :ref:`installation` section for more information about supported operating systems and toolchain.
+
+.. note::
+
+   Because of the significant, potentially breaking changes introduced in the |NCS| v2.7.0, the support for v2.7.99-cs1 in the |nRFVSC| is experimental in the following areas:
+
+     * sysbuild - During the extension stabilization period, manually edit and verify Kconfig option values instead of using the nRF Kconfig GUI.
+     * Hardware Model v2 - During the extension stabilization period, edit devicetree files manually instead of using the Devicetree Visual Editor.
+       Moreover, custom boards added in the extension will follow the Zephyr's Hardware Model v1.
+
+   The extension users that need v2.7.99-cs1 should `switch to the pre-release version of the extension`_.
+
+Migration notes
+***************
+
+See the `Migration notes for nRF Connect SDK v2.7.99-cs1 and the nRF54H20 DK`_ for the changes required or recommended when migrating your environment to the |NCS| v2.7.99-cs1.
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+Zephyr
+======
+
+.. NOTE TO MAINTAINERS: All the Zephyr commits in the below git commands must be handled specially after each upmerge and each nRF Connect SDK release.
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``ea02b93eea35afef32ebb31f49f8e79932e7deee``, with some |NCS| specific additions.
+
+* Added:
+
+  * Power management - Support for System On All Idle power state.
+  * Support to a new peripheral driver SAADC as experimental.
+  * Support for the Clock control API.
+
+For the list of upstream Zephyr commits (not including cherry-picked commits) incorporated into nRF Connect SDK since the most recent release, run the following command from the :file:`ncs/zephyr` repository (after running ``west update``):
+
+.. code-block:: none
+
+   git log --oneline ea02b93eea ^23cf38934c
+
+For the list of |NCS| specific commits, including commits cherry-picked from upstream, run:
+
+.. code-block:: none
+
+   git log --oneline manifest-rev ^ea02b93eea
+
+The current |NCS| main branch is based on revision ``ea02b93eea`` of Zephyr.
+
+.. note::
+   For possible breaking changes and changes between the latest Zephyr release and the current Zephyr version, refer to the :ref:`Zephyr release notes <zephyr_release_notes>`.
+
+Documentation
+=============
+
+* Added:
+
+  * A page about :ref:`ug_nrf54h20_architecture_clockman`.
+  * A page about :ref:`ug_nrf54h20_architecture_pm`.
+  * A :ref:`multicore_idle_test` sample documentation.
```

Comment:
This entry should move up in the samples section and should be modified into 'Added new sample .. ' under Other samples.

Reply:
In PR https://github.com/nrfconnect/sdk-nrf/pull/17015, changes are made only to the sample doc. Hence, documentation section could be the better place.